R/pkg/R/DataFrame.R

```diff
@@ -41,7 +41,7 @@ setOldClass("structType")
 #'\dontrun{
 #' sparkR.session()
 #' df <- createDataFrame(faithful)
-#'}
+#' }
```

Comment:
I'm not sure, when I scan through the R source files, I see both 'with one-space' and 'with no-space'.

so I should revert the adding space ones?